docs/pages/templates/unity.mdx

```diff
@@ -1,7 +1,7 @@
 # Unity
 
 [Unity](https://unity.com/) is a commercial game engine.
-[The integration is here](https://github.com/emergenceland/mud-template-unity).
+[The integration is here](https://github.com/emergenceland/UniMUD).
```

Comment:
didn't catch this earlier, but as a general rule, we should IMO avoid links that use "here" or "click here" etc.

see https://usability.yale.edu/web-accessibility/articles/links#link-text and https://heyoka.medium.com/dont-use-click-here-f32f445d1021 (there's many more articles on why, I just quickly grabbed these out of google)

```suggestion
The Emergence team has built [an open source Unity/MUD integration](https://github.com/emergenceland/UniMUD).
```

Reply:
@holic , done. Can you merge this? I'm not authorized, even when it is approved.